operator/deploy/crd.yaml

```diff
@@ -30,11 +30,335 @@ spec:
+                          properties:
+                            ca_file:
+                              type: string
+                            insecure_skip_verify:
+                              type: boolean
+                            password:
+                              type: string
+                            token:
+                              type: string
+                            type:
+                              type: string
+                            use_kiali_token:
+                              type: boolean
+                            username:
+                              type: string
+                        dashboards:
+                          type: array
+                          items:
+                            type: object
+                            additionalProperties: true
+                        enabled:
+                          type: boolean
+                        in_cluster_url:
+                          type: string
+                        url:
+                          type: string
+                    istio:
+                      type: object
+                      properties:
+                        istio_identity_domain:
+                          type: string
+                        istio_sidecar_annotation:
+                          type: string
+                        url_service_version:
+                          type: string
+                    prometheus:
+                      type: object
+                      properties:
+                        auth:
+                          type: object
+                          properties:
+                            ca_file:
+                              type: string
+                            insecure_skip_verify:
+                              type: boolean
+                            password:
+                              type: string
+                            token:
+                              type: string
+                            type:
+                              type: string
+                            use_kiali_token:
+                              type: boolean
+                            username:
+                              type: string
+                        custom_metrics_url:
+                          type: string
+                        url:
+                          type: string
+                    tracing:
+                      type: object
+                      properties:
+                        auth:
+                          type: object
+                          properties:
+                            ca_file:
+                              type: string
+                            insecure_skip_verify:
+                              type: boolean
+                            password:
+                              type: string
+                            token:
+                              type: string
+                            type:
```

Comment:
type should be an enum - it accepts 3 values - `basic`, `bearer`, and `none` - see: https://github.com/kiali/kiali/blob/v1.9/config/config.go#L47-L50